src/styles/ErrorModal.scss

```diff
@@ -0,0 +1,28 @@
+.error-modal {
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
```

Comment:
My weird trick for centering modals (if you don't want them to be full screen) is adding this `translate(-50%,-50%)`